SERVER/api/v1/models/adModel.js

```diff
@@ -40,6 +41,31 @@ class CarSaleAd{
         return queryResult;
 
     }
+
+    // Return all cars (sold & available)
+    async allCars(){
+        const queryText = 'SELECT * FROM ads'; 
+        const queryResult = await db.query(queryText) 
+        return queryResult;
+    }
+
+    //  Get a car's status
+    async availableCars(theAvailable){
+        const queryText = 'SELECT ads.car_id,  users.email AS owner, ads.manufacturer, ads.body_type,  ads.model, ads.state, ads.status, ads.price FROM ads INNER JOIN  users ON ads.owner=users.email  WHERE ads.status=$1'; 
+        const queryResult = await db.query(queryText, [theAvailable]); 
+        return queryResult; 
+    } 
+
+    async findPrice(){
```

Comment:
This is a repetition @Moise1 , We can combine `allCars()` and  `findPrice()` as a single method to avoid the repetition. The method can take in two parameters, 1 to replace `*` and `ads.price` and the other to replace `ads` or `table name`. eg
```
async getData(dataToQuery, tableName){
const queryText = 'SELECT dataToQuery FROM tableName'; 
.....
....
}

Reply:
@Kasulejoseph, I'm done applying the feedback on this branch.